src/accessibility.md

```diff
@@ -6,49 +6,128 @@ layout: layout-single-page-prose.njk
 
-GDS is committed to making its websites accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.
+The Equality and Human Rights Commission (<abbr>EHRC</abbr>) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’). If you contact us with a complaint and you’re not happy with our response, [contact the Equality Advisory and Support Service (<abbr>EASS</abbr>)](https://www.equalityadvisoryservice.com/).
 
 ### Compliance status
 
-The Design System website at [https://design-system.service.gov.uk/](/) is fully compliant with the Web Content Accessibility Guidelines (WCAG) version 2.1 AA standard.
+The Design System website at [design-system.service.gov.uk](https://design-system.service.gov.uk/) is partially compliant with the Web Content Accessibility Guidelines (<abbr>WCAG</abbr>) version 2.1 AA standard. Non-accessible content is documented in the next section.
+
+The GOV.UK Frontend documentation website at [frontend.design-system.service.gov.uk](https://frontend.design-system.service.gov.uk/) is fully compliant with the Web Content Accessibility Guidelines (<abbr>WCAG</abbr>) version 2.1 AA standard.
+
+The GOV.UK Frontend codebase at [github.com/alphagov/govuk-frontend](https://github.com/alphagov/govuk-frontend) is fully compliant with the Web Content Accessibility Guidelines (<abbr>WCAG</abbr>) version 2.1 AA standard. Non-accessible content is documented in the next section.
+
+### Non-accessible content
+
+In this section, we only list non-accessible content that has been reported, verified and tracked in the following repositories:
+
+- [GOV.UK Design System GitHub repository](https://github.com/alphagov/govuk-design-system)
+- [GOV.UK Frontend Documentation Github repository](https://github.com/alphagov/govuk-frontend-docs)
+- [GOV.UK Frontend GitHub repository](https://github.com/alphagov/govuk-frontend)
+
+For this section, we define an ‘accessibility concern’ as any question about the accessibility of a portion of one of our products.
+
+#### Non-compliance with the accessibility regulations
+
+These accessibility concerns have been classified as being non-compliant, due to a failure in one or more WCAG 2.1 Level AA criteria.
+
+WCAG 2.1 AA failures are documented in two of our repositories under the ‘accessibility-regulations-failure’ issue label:
+
+- [GOV.UK Design System website label for accessibility regulations failures](https://github.com/alphagov/govuk-design-system/labels/accessibility-regulations-failure)
+- [GOV.UK Frontend codebase label for accessibility regulations failures](https://github.com/alphagov/govuk-frontend/labels/accessibility-regulations-failure)
+
+From the GOV.UK Design System website:
```

Comment:
Just a suggestion: could these 3 areas be done as headers? I got confused a bit reading it, especially with the "current status" way it's written.

```suggestion
##### From the GOV.UK Design System website
```

Reply:
Hey @calvin-lau-sig7! Do we allow H5s in our content? I was only avoiding it because I for some reason thought we didn't use them.

Reply:
After a chat with Calvin and Claire, I think we'll leave these as paragraphs for now, and not go down to an H5 heading level. We don't have any styling set for H5 elements on the GOV.UK Design System website at the moment.

If it becomes confusing in the future, we can consider making headings.